### ISSUE TITLE
Support for splitting windows.  

### ISSUE THREAD

**Issue by Zamicol** [visualfc/liteide#602]
Netbeans has a nice feature for splitting windows.  The feature allows two sections of code to be viewed at once.  

It would be nice to also have this feature in liteide.  

![image](https://cloud.githubusercontent.com/assets/5232965/14765467/f3b055fc-099f-11e6-8ec9-a0d9d6e20890.png)

**Comment by xgdgsc** [comment]
LiteIDE feels like qtcreator for me, except this function!

**Comment by joncrlsn** [comment]
I love liteide!  It's small enough to run on my 15" Chromebook and it does everything I want in a Go IDE... except showing me two windows side-by-side or one on top of the other.

**Comment by Lida4py** [comment]
I agree with joncrlsn, showing windows side-by-side is an important feature for me and I really miss it in liteIDE :(

**Comment by rfjakob** [comment]
What you can already do is:

Right-Click the tab title -> "Move to new window"

And then move the windows side by side. Actually, this is good enough for me (screenshot attached).

![screenshot from 2019-01-01 16-06-12](https://user-images.githubusercontent.com/286847/50574004-3637b780-0ddf-11e9-9028-4177154f9746.png)

**Comment by widon1104** [comment]
这个功能很实用啊，上下分割，左右分割，3年了，还是没有动静
